pkg/spec/spec.go

```diff
@@ -39,6 +39,7 @@ type FunctionSpec struct {
 	Deps     string            `json:"deps"`
 	Memory   resource.Quantity `json:"memory"`
 	Env      map[string]string `json:"env"`
+	Desc     string            `json:"description"`
```

Comment:
I'm sort of ok with this, but it's odd having a `description` field (none of the standard kubernetes objects have it).   If we're only adding it because serverless wants it(?), then an alternative would be to just add it as a k8s annotation (or even label) in the serverless client - this is the sort of thing that the annotations extension mechanism is there for.